ui/src/message_popup/scoring_question_context.jsx

```diff
@@ -38,7 +38,7 @@ export default React.createClass({
             expandable={true}>
               <div>
                 <div style={{paddingBottom: 20}}>{question.text}</div>
-                {question.student && <StudentCard useCardStyles={true} student={question.student} />}
+                {question.students && question.students.map(student => <StudentCard useCardStyles={true} student={student} />)}
```

Comment:
Should this be wrapped in a `div` to hold all the students, or should this have a `key` for each `StudentCard` in the list?